src/styles/Card.js

```diff
@@ -8,6 +8,7 @@ const Card = styled.div`
   border: 0 solid rgba(123, 123, 123, 0.498039);
   border-radius: ${borderRadius};
   font-size: ${size.small};
+  font-family: "Roboto";
```

Comment:
This existing at a higher level: https://github.com/bdougie/open-sauced/blob/master/src/index.css#L84